packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -1089,8 +1090,11 @@ export class InputDatePicker
         )
       : null;
 
-    this.setInputValue((date && this.dateTimeFormat.format(date)) ?? "", "start");
-    this.setInputValue((this.range && endDate && this.dateTimeFormat.format(endDate)) ?? "", "end");
+    this.setInputValue((date && this.dateTimeFormat?.format(date)) ?? "", "start");
```

Comment:
By calling `handleDateTimeFormatChange` early in the lifecycle, isn't `dateTimeFormat` guaranteed?